gulp/browser-test.js

```diff
@@ -40,6 +40,6 @@ function browserWatch() {
   gulp.watch(['src/**/*.js', 'test/**/*.js'], ['browser-bundle']);
 }
 
-gulp.task('browser-bundle', ['lint-src', 'lint-test'],   bundle);
+gulp.task('browser-bundle', [],   bundle);
```

Comment:
async would be good. My behavior is to run both the browser and terminal tests so that i can quickly see if there's something for me to investigate in the browser... Could be weird...